install/copy_ants_binaries.py

```diff
@@ -43,48 +38,41 @@ def usage():
 #=======================================================================================================================
 
 folder_bin = 'bin'
-prefix_sct = 'sct_'
+prefix_sct = 'isct_'
 listOS = ['osx', 'linux']
-list_file_ants = ['antsApplyTransforms', 'antsRegistration', 'antsSliceRegularizedRegistration', 'ANTSUseLandmarkImagesToGetAffineTransform', 'ANTSUseLandmarkImagesToGetBSplineDisplacementField', 'ComposeMultiTransform', 'ImageMath', 'ThresholdImage']
-OSname = ''
+list_file_ants = ['antsApplyTransforms', 'antsRegistration', 'antsSliceRegularizedRegistration', 'ComposeMultiTransform']
+output_folder = 'antsbin/'
 
-# get path of SCT
-status, path_sct = commands.getstatusoutput('echo $SCT_DIR')
 
 # Check input param
 try:
-    opts, args = getopt.getopt(sys.argv[1:],'hf:s:')
+    opts, args = getopt.getopt(sys.argv[1:],'hf:')
 except getopt.GetoptError as err:
-    print str(err)
+    print(str(err))
     usage()
 if not opts:
     usage()
 for opt, arg in opts:
     if opt == '-h':
         usage()
-    elif opt in ('-s'):
-        OSname = str(arg)
     elif opt in ('-f'):
-        folder_ants = sct.slash_at_the_end(str(arg), 1)  # add slash at the end
-
-# check if OS exists
-if OSname not in listOS:
-    sct.printv('\nERROR: OS name should be one of the following: '+'[%s]' % ', '.join(map(str, listOS))+'\n', 1, 'error')
+        folder_ants = str(arg)+'/bin/'
 
 # check if ANTs folder exists
 if not os.path.exists(folder_ants):
-    sct.printv('\nERROR: Path '+folder_ants+' does not exist.\n', 1, 'error')
+    print('\nERROR: Path '+folder_ants+' does not exist.\n')
 
 # build destination path name
-path_bin = sct.slash_at_the_end(path_sct, 1)+folder_bin+'/'+OSname+'/'
+os.makedirs(output_folder)
 
 # loop across ANTs files
 for file_ants in list_file_ants:
     # check if file exists
     if not os.path.exists(folder_ants+file_ants):
-        sct.printv('\nERROR: File '+folder_ants+file_ants+' does not exist.\n', 1, 'error')
+        print('\nERROR: File '+folder_ants+file_ants+' does not exist.\n')
     # copy data
-    sct.run('cp '+folder_ants+file_ants+' '+path_bin+prefix_sct+file_ants, 1)
-
-print "done!\n"
+    print('Copying: '+file_ants)
```

Comment:
For complete compatibility with all version of python, the future import is required `from __future__ import print_function` as the first import statement.

https://stackoverflow.com/questions/32032697/how-to-use-from-future-import-print-function